### PR TITLE
Reorganize the code before starting 1.0 work

### PR DESCRIPTION
**Pull request by ericmvaughn**

### CODE REVIEW

File: initialize.js [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+  if (err) {
+    console.log('error getting POE chaincodeID ' + err);
+  }
+  if (value) {
+    console.log('POE chaincode ID is ', value);
+    poeChaincode.id = value.trim();
+  }
+});
+
+function newUser(appUser) {
```
Comment (lee-merrill) [inline initialize.js]: Should this be in util.js instead of initialize.js?

File: initialize.js [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+        appUser.eSignKey = record.key;
+        appUser.eSignCallbackKey = record.callbackKey;
+      } catch (e) {
+        console.log('Error parsing eSignLive record ', e);
+      }
+    }
+  });
+
+  // register and enroll a new user
+  chain.getUser(appUser.userName, function(err, user) {
```
Comment (lee-merrill) [inline initialize.js]: Maybe put this in util.js?

File: signNowRouter.js [hunk 3]
```diff
@@ -0,0 +1,515 @@
...
+// jscs:disable requireCamelCaseOrUpperCaseIdentifiers
+// Preform the event subscription for the given user
+router.post('/subscribe', function(req, res) {
+  let name = req.body.name;
+  if (!req.session.appUser) {
+    console.log('No user set');
+    return res.status(500).send('No user set.');
+  }
+  var appUser = GlobalAppUser[req.session.appUser];
+  if (!appUser.snId || !appUser.snToken.access_token) {
```
Comment (lee-merrill) [inline signNowRouter.js]: Should be '|| !appUser.snToken || !appUser.snToken.access_token'?

**Comment by lee-merrill** [comment]
Looks good...

File: initialize.js (hunk 1 above)
Comment (ericmvaughn) [inline initialize.js]: Done

File: initialize.js (hunk 2 above)
Comment (ericmvaughn) [inline initialize.js]: Done

File: signNowRouter.js (hunk 3 above)
Comment (ericmvaughn) [inline signNowRouter.js]: changing it to 
  if (!appUser.snId || !appUser.snToken || !appUser.snToken.access_token) {

**Comment by lee-merrill** [comment]
Changes look good...

**Pull request merged**